apps/vs-code-designer/src/app/commands/deploy/deploy.ts

```diff
@@ -235,12 +241,15 @@ async function getDeployLogicAppNode(context: IActionContext): Promise<SlotTreeI
     return await createLogicApp(context, sub);
   }
 
+  let secrets: ContainerAppSecret[] = [];
+
   if (site.id.includes('Microsoft.App')) {
     // NOTE(anandgmenon): Getting latest metadata for hybrid app as the one loaded from the cache can have outdateed definition and cause deployment to fail.
     const clientContainer = await createContainerClient({ ...context, ...sub.subscription });
     site = (await clientContainer.containerApps.get(site.id.split('/')[4], site.name)) as undefined as Site;
+    secrets = (await clientContainer.containerApps.listSecrets(site.id.split('/')[4], site.name)).value;
```

Comment:
nit: can we name this `site.id.split('/')[4]` as someone who doesn't know this feature very well it would be good to have more context on how this works